packages/select/src/components/query-list/queryList.tsx

```diff
@@ -34,12 +34,6 @@ import {
 export type QueryListProps<T> = IQueryListProps<T>;
 /** @deprecated use QueryListProps */
 export interface IQueryListProps<T> extends ListItemsProps<T> {
-    /** If provided, this function will be used in place of the default implementation. */
-    getActiveElement?: (props: {
```

Comment:
I moved this into `ListItemProps` so that users can pass this to components like `MultiSelect2`